include/cantera/thermo/Elements.h

```diff
@@ -10,6 +10,8 @@
 
 #include "cantera/base/ct_defs.h"
 
+using namespace std;
```

Comment:
If I remove this line, I get compiler errors for every file that indirectly includes Elements.h:

    include/cantera/thermo/Elements.h:115:1: error: 'string' does not name a type
     string getElementSymbol(const std::string& ename);

This is on Ubuntu 14.04 with gcc 4.8.4. My cantera.conf is

    python3_package = 'n'
    f90_interface = 'n'
    system_sundials = 'n'

How should I handle that? You said you couldn't reproduce the error, but this is definitely repeatable for me.